setup.py

```diff
@@ -12,7 +12,7 @@
 
 setup(
     name="RAstral Rampart",
-    version="1.0.0",
+    version="1.0.1",
```

Comment:
```suggestion
    version="1.1.0",
```
minor but I think this is the right versioning for this sort of change